Source/WebCore/page/Screen.cpp

```diff
@@ -92,7 +92,7 @@ unsigned Screen::colorDepth() const
 {
     RefPtr frame = this->frame();
     if (!frame)
-        return 0;
+        return 24;
```

Comment:
I wonder if it's worth codifying this default value as

```cpp
static constexpr uint32_t CSSOMDefaultColorDepth = 24;
```
in `Screen.h` and sharing between `Screen::colorDepth` and `PlatformScreenIOS::colorDepth`?

Reply:
The iOS value is really the platform value (though it's a constant for "reasons") and this is the universal fallback value so I think keeping them distinct makes sense.

However, if `colorDepth` is not an adequate API long term for HDR and such we might consider always returning 24 and not having the platform implementations of this API.